bouncer/shared/utils.ts

```diff
@@ -499,7 +499,9 @@ export async function observeSwapRequested(
           (typeof id === 'number' &&
             'DepositChannel' in data.origin &&
             Number(data.origin.DepositChannel.channelId.replaceAll(',', '')) === id) ||
-          (typeof id === 'string' && 'Vault' in data.origin && data.origin.Vault.txHash === id);
+          (typeof id === 'string' &&
+            'Vault' in data.origin &&
+            data.origin.Vault.txHash.ByteHash === id);
```

Comment:
FYI: For now, it's safe to assume that we always have to deal with EVM or BTC, so it will be always a H256 hash string. If this should change, we have to handle this somehow.

Reply:
We have the Solana Vault swaps PR in the pipeline that will definitely break that. I'll make sure to update that after this is complete. Thanks for pointing that out.